scipy/optimize/tests/test_linprog.py

```diff
@@ -1822,6 +1909,26 @@ class TestLinprogRSBland(LinprogRSTests):
     options = {"pivot": "bland"}
 
 
+############################################
+# HiGHS-Simplex-Dual Option-Specific Tests #
+############################################
+
+
+class TestLinprogHiGHSSimplexDual(LinprogHiGHSTests):
+    method = "highs-ds"
+    options = {}
```

Comment:
No tests with non-default options needed? There's quite a few HiGHS-specific options right? Fine with me to leave for later, just want to check cause it looks a bit sparse.

Reply:
I'll let @mdhaber weigh on option coverage

Reply:
In the common suite, there are tests for non-default `maxiter`, `disp`, and `presolve` - the common options. We could add tests that check that non-default values of method-specific options don't break things, but I was having a hard time coming up with stronger tests, so I guess I didn't add them. We did remove most of the HIGHS-specific options that we had, actually; it's down to tolerances, time limit, and simplex pivot strategy. I'll think about adding something when fixing merge conflicts, but I agree that it would be OK to leave for later.

Reply:
added to linprog cleanup list in gh-9671

Reply:
> added to linprog cleanup list in gh-9671

sounds good, resolving